var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -187,3 +188,175 @@ def cmake_args(self):
                 options.append("-DVTKm_ENABLE_TBB:BOOL=OFF")
 
             return options
+
+    @run_after('install')
+    @on_package_attributes(run_tests=True)
+    def check_install(self):
+        # default post installation check
+        # do nothing for now
+        pass
```

Comment:
We don't need this right?

Reply:
I want to have a rule so that the test only occurs when building the master with 'run_tests' turned on.  However, it gives me an error if we do not have this dummy test function when building on other cases (where we don't want test). For example, if we build the default version 1.5.1 (not master) without having this dummy function, it produces an error: 

> Error: NoSuchMethodError: Package VtkM does not support check_install_master called with vtk-m@1.5.1

Maybe there is a better way to set up this rule?

Reply:
Never mind. I fixed that after reordered the decorators. The dummy function is removed.